### PR TITLE
feat(faq-page): initial-setup

### PR DESCRIPTION
**Pull request by PatrickDohn**
#### Summary

- Work Type:  Feature 

#### What I did

- setup faq page
-
-

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in format `TYPE(scope): description`
- [x] Commits are in semantic format
- [x] Has Summary
- [x] Has Labels

### CODE REVIEW

File: src/scss/FAQPage.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.faq-page {
```
Comment (davidholyko) [inline src/scss/FAQPage.scss:1]: re order selectors

```
faq 
faq-link
faq-<whatever comes next>

```

File: src/components/pages/FAQPage.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          </div>
+          <div className="faq-questions">
+            <h1>Who is the project for?</h1>
+            <h3>
+              Software engineers coming out of a bootcamp who know some CSS and
+              JSON.
+            </h3>
+          </div>
+          <div className="faq-questions">
+            <h1>How do i find the code for this project?</h1>
```
Comment (davidholyko) [inline src/components/pages/FAQPage.js:28]: capitalize i

File: src/components/pages/FAQPage.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <main className="faq-page">
+      <NavBar />
+      <div className="faq">
+        <section className="faq-section">
+          <h1 className="faq-title">FAQ</h1>
+          <div className="faq-questions">
+            <h1>What technologies are used to build this project</h1>
+            <h3>
+              ResGenie is built with React, with a custom webpack config. The
```
Comment (davidholyko) [inline src/components/pages/FAQPage.js:15]: "ResGenie is built with React. It's JSON editor is made from codemirror (add link to codemirror website) and the PDF iframe is built with react-pdf (add link to https://react-pdf.org/). Please check out a more comprehensive list here (link to https://github.com/starter-code/res-gen#technology)

**Review by davidholyko — CHANGES_REQUESTED** [review]
It looks like theres some merge conflicts. revert package lock deletion and index.scss conflict.

Probably youll have to rebase

File: src/scss/FAQPage.scss (hunk 1 above)
Comment (PatrickDohn) [reply src/scss/FAQPage.scss:1]: Taken care of

File: src/components/pages/FAQPage.js (hunk 2 above)
Comment (PatrickDohn) [reply src/components/pages/FAQPage.js:28]: Taken care of

File: src/components/pages/FAQPage.js (hunk 3 above)
Comment (PatrickDohn) [reply src/components/pages/FAQPage.js:15]: Taken care of

File: src/scss/index.scss [hunk 4]
```diff
@@ -9,6 +9,7 @@
 @import './Modal.scss';
 @import './NavBar.scss';
 @import './Footer.scss';
+@import './FAQPage.scss';
```
Comment (davidholyko) [inline src/scss/index.scss:12]: import from main seems to be lost in the rebase.

https://github.com/starter-code/res-gen/blob/master/src/scss/index.scss#L12

**Review by davidholyko — APPROVED** [review]
LGTM

**Pull request merged**